### PR TITLE
Add notes on prepping a new release series

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
LGTM. Let's keep SOFTWARE-2622 open until I put some more content into this.

**Comment by brianhlin** [comment]
Do you have specific things you have in mind? Otherwise, I think this should be enough to go off of in the future.

**Comment by matyasselmeci** [comment]
Yeah, actual code, not just a TODO list.

**Comment by brianhlin** [comment]
Does that code already exist? In other words, is this a 30 min collation job?

**Comment by matyasselmeci** [comment]
Exist? Yes. 30 min? No.

**Comment by brianhlin** [comment]
It's an internal doc, it doesn't have to be your magnum opus. Can someone reasonably intelligent use what you already have as a template/starting point and apply it to subsequent releases? New release series are infrequent enough that you should keep [this chart](https://xkcd.com/1205/) in mind. Plus, we can always improve the content later down the line. 

tl;dr If copying what you have with a little bit of explanation text takes you an ~hour, let's just get it done.
